### DISCUSSION TITLE
RClone mounted Google Photos

### DISCUSSION THREAD

**Discussion by latetedemelon** [simulot/immich-go#17] [Q&A]
@simulot - Love the project, this is exactly what I was looking for.   I was set up to do a different way, and mounted Google Photos in a read only directory via Rclone.  Does your script support this scenario?  

photos
|-- album
|   |-- albums
|-- feature
|   |-- favourites
|-- media
|   |-- all
|   |-- by-day
|   |-- by-month
|   |-- by-year
|-- shared-album
|   |-- shared albums
|-- upload

**Comment by simulot** [comment]
> @simulot -Love the project, this is exactly what I was looking for.

Thanks!

How do you get this tree? What RClone does?
Are those folder imported as a read only gallery?
Could you describe your idea?

**Reply by latetedemelon** [reply]
Rclone.org - allows you to to control / update remote cloud storage from the command line.  I simply used it to mount my Google Photos account to a specific directory (read only).

**Reply by simulot** [reply]
I didn't know it was possible to mount a google photo storage using rclone.

The last version of immich server provides a new feature for libraries (https://immich.app/docs/features/libraries) that may goes in your direction. You may need to call the api to force a scan.

**Reply by ne0ark** [reply]
I am also interested in this approach. I used Rclone to mount Google Photos. Is there a way to import images into a Immich? External libraries don't work because I want to move the images, not store them in their current location.
